hack/generated/controllers/generic_controller.go

```diff
@@ -333,9 +332,13 @@ func (gr *GenericReconciler) StartDeleteOfResource(
 	}
 
 	// delete has started, check back to seen when the finalizer can be removed
-	return ctrl.Result{
-		RequeueAfter: gr.RequeueDelay,
-	}, nil
+	requeueDelay := gr.RequeueDelay
+	if retryAfter > requeueDelay {
+		data.log.Info("DELETE started, will check again", "delay", retryAfter/time.Second)
```

Comment:
TODO should probably remove these logs

Reply:
Or at least make their log level really high

Reply:
Moved to log level 3, since at 4 we dump JSON 🙂